kobo/apps/subsequences/management/commands/migrate_submission_supplements.py

```
@@ -107,36 +105,59 @@ def handle(self, *args, **options):
+                to_update.append(ss)
+                migrated += 1
+            else:
+                # check for mistakes in manual_qual versions from a previous migration
+                update_necessary = False
+                for question_xpath, action_results in content.items():
+                    if question_xpath == '_version':
+                        continue
+                    for action, action_responses in action_results.items():
+                        if action != 'manual_qual':
+                            continue
+                        for question_uuid, versions_data in action_responses.items():
```

Ruff (0.15.10)

[warning] Loop control variable `question_uuid` not used within loop body

Rename unused `question_uuid` to `_question_uuid`

(B007)